auth/cookie-auth/src/main.rs

```diff
@@ -33,16 +33,16 @@ async fn main() -> std::io::Result<()> {
     std::env::set_var("RUST_LOG", "actix_web=info");
     env_logger::init();
 
-    // Generate a random 32 byte key. Note that it is important to use a unique
-    // private key for every project. Anyone with access to the key can generate
-    // authentication cookies for any user!
-    let private_key = rand::thread_rng().gen::<[u8; 32]>();
+    /// Generate a random secret key. Note that it is important to use a unique
+    /// secret key for every project. Anyone with access to the key can generate
+    /// authentication cookies for any user!
```

Comment:
Doc comment doesn't make sense here.

```suggestion
    // Generate a random secret key. Note that it is important to use a unique
    // secret key for every project. Anyone with access to the key can generate
    // authentication cookies for any user!
```